source/modules/MEI.cmn.xml

```diff
@@ -452,6 +452,15 @@
       </attDef>
     </attList>
   </classSpec>
+ <classSpec ident="att.damp.log" module="MEI.cmn" type="atts">
+    <desc xml:lang="en">Logical domain attributes.</desc>
+    <classes>
+      <memberOf key="att.controlEvent"/>
+      <memberOf key="att.duration.additive"/>
+      <memberOf key="att.startEndId"/>
+      <memberOf key="att.timestamp2.log"/>
+    </classes>
+  </classSpec>
```

Comment:
This should be moved ahead of `att.expandable` (for alphabetic order `att.mNum.log` needs to be ignored for now and like att.mDiv,... in the other files be moved to m later)

Reply:
@musicEnfanthen, I moved `att.damp.log`.  I don't know what you're trying to say about `att.mNum.log` and `att.mDiv.log`.  Why should they not be moved now?

Reply:
`att.mNum.log` and `att.mDiv.log` are in wrong alphabetical order. I guess I wanted to say that we should fix this in another PR, not in this one, since it has nothing to do with the string damp feature.

Reply:
Marking this as resolved.  I'll make another pull request to change the order of declarations.